.env

```diff
@@ -0,0 +1,7 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
`.env` file shouldn't be committed as it contains sensitive info like our db key